.github/workflows/lmi_dist_integration.yml

```diff
@@ -2,6 +2,11 @@ name: LMI Dist library Unit/Integration tests
 
 on:
   workflow_dispatch:
+    inputs:
```

Comment:
since we anyway run the test with the full model server, should we just merge with @sindhuvahinis 's pipeline?

Reply:
Done